tools/waveica/waveica_wrapper.R

```diff
@@ -147,9 +211,12 @@ exclude_group <- function(data, group) {
   }
 }
 
-
-# Store output of WaveICA in a tsv file
-store_data <- function(data, output) {
-  write.table(data, file = output, sep = "\t", row.names = FALSE, quote = FALSE)
-  cat("Normalization has been completed.\n")
+store_data <- function(data, output, ext) {
+  if (ext == "csv") {
+    write_csv(data, output)
+  } else if (ext == "tsv") {
+    write_tsv(data, output)
+  } else {
+    write_parquet_file(data, output)
+  }
```

Comment:
I think the functions from above can be all put here, since otherwise they are 1 line functions.

Reply:
resolved in [eb327e3](https://github.com/RECETOX/galaxytools/pull/304/commits/eb327e3d89c96129f704a27e56f07ace5b90e8af)